.nvmrc

```diff
@@ -0,0 +1 @@
+10.11.0
```

Comment:
Why is this needed actually? Is there a package we use that require at least this version of node? Is it redundant with what you added in package.json?

Reply:
In the Electron [Release Notes for v4.0.0](https://github.com/electron/electron/releases/tag/v4.0.0) it says: 

> Bumped minimum supported macOS version to 10.10.

However the release notes for v4.0.1 or v3.1.0 don't mention a minimum version requirement https://github.com/electron/electron/releases